### ISSUE TITLE
f10_f10_musgs grid doesn't work with NUOPC

### ISSUE THREAD

**Issue by ekluzek** [ESCOMP/CTSM#1543]
### Brief summary of bug

The USGS mask doesn't work in NUOPC, so tests with f10_f10_musgs fail when NUOPC is the default driver.

### General bug information

**CTSM version you are using:** ctsm5.1.dev061-37-g6c4fb1ef1 (preliminary ctsm5.1.dev062)

**Does this bug cause significantly incorrect results in the model's science?** No

**Configurations affected:** musgs mask

### Details of bug

The USGS mask wasn't really setup in a consistent way in the system and hasn't been ported over to NUOPC. So the mg37 mask should be used instead.

We realized this was an issue earlier, and switched tests out, but this must have come in after that change.

### Important details of your setup / configuration so we can reproduce the bug

We have two tests on cheyenne that show this problem...

SMS_D_Ld5.f10_f10_musgs.I2000Clm50FatesRs.cheyenne_gnu.clm-FatesColdDef		
SMS_D_Ld5.f10_f10_musgs.I2000Clm50FatesRs.cheyenne_intel.clm-FatesColdDef

There's also one on izumi_nag, and the cheyenne_gnu is in the prebeta test list

### Important output or errors that show the problem

```
Command: /glade/work/erik/ctsm_worktrees/external_updates/cime/scripts/create_newcase --case /glade/scratch/erik/tests_ctsm51d62acl/SMS_D_Ld5.f10_f10_musgs.I2000Clm50FatesRs.cheyenne_gnu.clm-FatesColdDef.GC.ctsm51d62acl_gnu --res f10_f10_musgs --compset I2000Clm50FatesRs --test --machine cheyenne --compiler gnu --project P93300606  --output-root /glade/scratch/erik/tests_ctsm51d62acl  --workflow default --user-mods-dir /glade/work/erik/ctsm_worktrees/external_updates/cime_config/testdefs/testmods_dirs/clm/FatesColdDef  --queue=premium --walltime 00:20:00
Output: Compset longname is 2000_DATM%GSWP3v1_CLM50%FATES_SICE_SOCN_SROF_SGLC_SWAV
Compset specification file is /glade/work/erik/ctsm_worktrees/external_updates/cime_config/config_compsets.xml
Automatically adding SESP to compset
Traceback (most recent call last):
  File "/glade/work/erik/ctsm_worktrees/external_updates/cime/scripts/create_newcase", line 265, in <module>
    _main_func(__doc__)
  File "/glade/work/erik/ctsm_worktrees/external_updates/cime/scripts/create_newcase", line 257, in _main_func
    ngpus_per_node=ngpus_per_node)
  File "/glade/work/erik/ctsm_worktrees/external_updates/cime/scripts/Tools/../../scripts/lib/CIME/case/case.py", line 1795, in create
    ngpus_per_node=ngpus_per_node)
  File "/glade/work/erik/ctsm_worktrees/external_updates/cime/scripts/Tools/../../scripts/lib/CIME/case/case.py", line 974, in configure
    gridinfo = grids.get_grid_info(name=grid_name, compset=self._compsetname, driver=self._comp_interface)
  File "/glade/work/erik/ctsm_worktrees/external_updates/cime/scripts/Tools/../../scripts/lib/CIME/XML/grids.py", line 80, in get_grid_info
    domains  = self._get_domains(component_grids, atmlevregex, lndlevregex, driver)
  File "/glade/work/erik/ctsm_worktrees/external_updates/cime/scripts/Tools/../../scripts/lib/CIME/XML/grids.py", line 219, in _get_domains
    mask_domain_node = self.get_optional_child("domain", attributes={"name":domains["MASK_GRID"]},
KeyError: 'MASK_GRID'
```

**Comment by mvertens** [comment]
@ekluzek - this is really not a bug. We have previously decided that we will not be supporting musgs at this point and only use the mg37 ocean mask. We had removed all references os musgs from the CTSM test suite - so I'm not sure why this is back.

**Comment by ekluzek** [comment]
Yes, I agree it's a limitation that we decided we are fine with. And as I mention in the issue the USGS mask wasn't setup in a consistent manner to other land/ocean masks. I don't see another label to call it by right now. But, I'll suggest we add "limitation" as a label later.

This was just recently added after we removed the musgs versions, and I'd call it an accident. I think it's just because f10_f10_musgs had been the grid used for a long time. With a larger project it's hard to make sure everyone remembers everything that's going on. We'll talk about it at our software meeting, but I'm not sure what could have been done differently to prevent it.
